src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incentives/IncentivesService.kt

```diff
@@ -265,28 +265,25 @@ class IncentivesService(
       defaultOnAdmission = prisonIncentiveLevel.default,
       active = prisonIncentiveLevel.active,
       remandSpendLimitInPence = prisonIncentiveLevel.remandSpendLimit?.let {
-        prisonIncentiveLevel.remandSpendLimit.multiply(
-          BigDecimal.valueOf(100),
-        ).toInt()
+        prisonIncentiveLevel.remandSpendLimit.toPence()
       },
       remandTransferLimitInPence = prisonIncentiveLevel.remandTransferLimit?.let {
-        prisonIncentiveLevel.remandTransferLimit.times(
-          BigDecimal.valueOf(100),
-        ).toInt()
+        prisonIncentiveLevel.remandTransferLimit.toPence()
       },
       convictedSpendLimitInPence = prisonIncentiveLevel.convictedSpendLimit?.let {
-        prisonIncentiveLevel.convictedSpendLimit.times(
-          BigDecimal.valueOf(100),
-        ).toInt()
+        prisonIncentiveLevel.convictedSpendLimit.toPence()
       },
       convictedTransferLimitInPence = prisonIncentiveLevel.convictedTransferLimit?.let {
-        prisonIncentiveLevel.convictedTransferLimit.times(
-          BigDecimal.valueOf(100),
-        ).toInt()
+        prisonIncentiveLevel.convictedTransferLimit.toPence()
       },
     )
   }
 
+  private fun BigDecimal.toPence(): Int = this.movePointRight(2).toInt()
+  private fun Int.toPounds(): BigDecimal = this.div(
+    100.toFloat(),
+  ).toBigDecimal()
```

Comment:
sorry, i forgot to mention you could use movePointLeft here!

Reply:
There wasn’t an equivalent - I checked!